core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1344,6 +1351,51 @@ write_files:
         - kind: ServiceAccount
           name: heapster
           namespace: kube-system
```

Comment:
A blank line here would be nice to make it easier to see the boundaries of each file.